src/generate-trigger-properties/generate.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-relative-packages */
+import fs from "fs"
+import triggerConfig from "../../output-data/data/excluded-trigger-config.json"
+import type { ExcludedTriggerConfig, TriggerCode } from "../../output-data/types/types"
+
+const typedTriggerConfig = <ExcludedTriggerConfig>triggerConfig
+
+const generateForceConfig = (
```

Comment:
Should this not be `generateTriggerProperties`, to be in keeping with the folder structure?

Reply:
👍

Reply:
Actually, looking at it, this function is only generating the config for a single force so I think the naming is OK